filebeat/input/filestream/internal/task/group_test.go

```diff
@@ -67,6 +67,7 @@ func TestNewGroup(t *testing.T) {
 }
 
 func TestGroup_Go(t *testing.T) {
+	t.Skip("Flaky tests: https://github.com/elastic/beats/issues/4121")
```

Comment:
Looks like this link is incorrect.

Reply:
```suggestion
	t.Skip("Flaky tests: https://github.com/elastic/beats/issues/41218")
```

Reply:
Thanks for catching this copy/paste oops!